akka-actor/src/main/scala/akka/event/Logging.scala

```diff
@@ -1289,13 +1285,17 @@ trait LoggingAdapter {
     case Logging.WarningLevel ⇒ isWarningEnabled
     case Logging.InfoLevel    ⇒ isInfoEnabled
     case Logging.DebugLevel   ⇒ isDebugEnabled
+    case _                    ⇒ false
```

Comment:
Being the paranoid person I am I might've moved this behaviour (the match) in, like, a method in `LogLevel` just to ensure that if a new log level is added we very probably tweak this definition.